File: BlockBattleBot/Brain.cs
using System;
using System.Collections.Generic;

namespace BlockBattleBot
{
    public class Brain
    {
        public Game Game { get; private set; }

        public Brain(Game game)
        {
            Game = game;
        }

        public Move[] Moves(int time)
        {
            // AI logic goes here

            return new Move[] { Move.Drop };
        }
    }
}

File: src/Brain.ts
import { Game } from "./Game";
import { Move } from "./Move";

export class Brain {
	private _game: Game;
	
	constructor(game: Game) {
		this._game = game;
	}
	
	public moves(time: number): Array<Move> {
		return [Move.Drop];
	}
}